nymea-app/ui/mainviews/WallboxView.qml

```diff
@@ -46,70 +46,94 @@ MainViewBase {
         shownInterfaces: ["evcharger"]
     }    
 
-    SwipeView {
-        id: swipeView
+    Rectangle {
         anchors.fill: parent
-        anchors.horizontalCenter: parent.horizontalCenter
-        currentIndex: pageIndicator.currentIndex
-        visible: wallboxDevices.count != 0
+        color: app.foregroundColor
 
-        Repeater {
+        SwipeView {
+            id: swipeView
             anchors.fill: parent
             anchors.horizontalCenter: parent.horizontalCenter
```

Comment:
anchors.fill:parent makes the item to be sized and positioned same as the parent. This somewhat implies centering it already